### ISSUE TITLE
Tracker failing

### ISSUE THREAD

**Issue by DiegoRBaquero** [OpenWebTorrent/openwebtorrent-www#3]
WebSocket connection to 'wss://tracker.openwebtorrent.com/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED

**Comment by alxhotel** [comment]
Thanks ! Looks like it got this weird error and then stopped working

```
buffer.js:188
        throw new TypeError('First argument needs to be a number, ' +
              ^
TypeError: First argument needs to be a number, array or string.
    at new Buffer (buffer.js:188:15)
    at Object.exports.hexToBinary (/tracker/node_modules/bittorrent-tracker/lib/common.js:15:10)
    at Server._onWebSocketRequest (/tracker/node_modules/bittorrent-tracker/server.js:322:25)
    at WebSocket.EventEmitter.emit (events.js:98:17)
    at Receiver.ontext (/tracker/node_modules/bittorrent-tracker/node_modules/ws/lib/WebSocket.js:8$
    at /tracker/node_modules/bittorrent-tracker/node_modules/ws/lib/Receiver.js:477:18
    at /tracker/node_modules/bittorrent-tracker/node_modules/ws/lib/Receiver.js:361:7
    at /tracker/node_modules/bittorrent-tracker/node_modules/ws/lib/PerMessageDeflate.js:247:5
    at afterWrite (_stream_writable.js:274:3)
    at onwrite (_stream_writable.js:266:7)
```

**Comment by alxhotel** [comment]
Fixed in PR [#119](https://github.com/feross/bittorrent-tracker/pull/119)
